### DISCUSSION TITLE
Ignore some repos

### DISCUSSION THREAD

**Discussion by ryzmae** [anuraghazra/github-readme-stats#3041] [Q&A] [answered]
Hello ive cloned and delployed it to vercel but now its saying like 70% javascript because of the repo how can i ignore this repo

**Comment by rickstaa** [comment, marked as answer]
@ryzmae thanks for your requestion. As explained [in the documentation](https://github.com/anuraghazra/github-readme-stats#exclude-individual-repositories), this can be done through the `exclude_repo` variable:

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=ryzmae &exclude_repo=ryzmquiz)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=ryzmae&exclude_repo=ryzmquiz)](https://github.com/anuraghazra/github-readme-stats)
